### DISCUSSION TITLE
Question about the hashed block ID system.

### DISCUSSION THREAD

**Discussion by cen0b** [bedrock-tool/bedrocktool#113] [Q&A]
What is it, and what is the status on supporting it?

**Comment by olebeck** [comment]
its a new way the game encodes blocks in the world chunks, the issue is that i havent found documentation or code on what the hash function is

**Comment by Happy2018new** [comment]
I implement this few minutes ago, see https://github.com/YingLunTown-DreamLand/bedrock-world-operator/blob/main/block/hash.go for more information.

**Reply by olebeck** [reply]
I don't use the discussions tab so I forgot about this, implemented network hashed block IDs long ago

**Reply by Happy2018new** [reply]
> I don't use the discussions tab so I forgot about this, implemented network hashed block IDs long ago

You mean [this](https://github.com/bedrock-tool/bedrocktool/blob/master/subcommands/merge/block_registry.go#L21)?
However, I found nothing in dragonfly, and it seems like they don't support network block hash runtime id.
